src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
+
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
+
+        // Welcome Message
+        PrintWelcomeMessage();
+
+        // Active Chat
+        ActiveChat();
+
+        // Goodbye Message
+        PrintGoodbyeMessage();
+    }
+
+    private static void ActiveChat() {
+        boolean isBye = false;
+        boolean isEmptyList = true;
+        String input;
+        Scanner in = new Scanner(System.in);
+        int counter = 0;
+        Task[] list = new Task[100];
+
+        while(!isBye){
+            //store input
+            input = in.nextLine();
+            //check if input is empty
+            CheckIsEmptyList(list);
+            //process input
+            if (input.equals("bye")){ //check if bye
+                isBye = true;
+            } else if (input.equals("list")) { //check if list
+                ProcessList(counter, list);
+            } else if (input.contains("done") ) { //check if done
+                ProcessDone(input, list);
+            } else {
+                counter = ProcessTasks(input, counter, list); //process tasks
+            }
+        }
+    }
+
+    private static void CheckIsEmptyList(Task[] list) {
```

Comment:
'checkIsEmptyList' for 'CheckIsEmptyList', or perhaps even just 'isEmptyList' for boolean naming.
Similar camelCase method naming issues in other places as well.